packages/instant-meilisearch/src/client/instant-meilisearch-client.ts

```diff
@@ -98,7 +98,7 @@ export function instantMeiliSearch(
           const adaptedSearchResponse = adaptSearchResponse<T>(
             searchResponse,
             searchContext,
-            initialFacetDistribution[searchRequest.indexName]
+            initialFacetDistribution[searchContext.indexUid]
```

Comment:
No test?! 👀

Reply:
From my modest understanding of what's happening, it looks like the correct thing to do to retrieve the initial facet distribution.

Reply:
Done!